packages/instance-model/index.d.ts

```diff
@@ -1,12 +1,13 @@
 import type Collection from 'mongodb-collection-model';
 import type { DataService } from 'mongodb-data-service';
 import type { Collection as DatabaseCollection } from 'mongodb-database-model';
-
-import { ServerType } from './server-type';
-import { TopologyType } from './topology-type';
 import Database from 'mongodb-database-model';
 import { CollectionCollection } from 'mongodb-collection-model';
 
+// TODO: add real types for these
```

Comment:
nit: should we add a ticket?

Reply:
I'll open a follow-up right away! Didn't want to make the amount of required changes here bigger than needed